web/src/lib/components/asset-viewer/photo-viewer.svelte

```diff
@@ -6,12 +6,14 @@
   import { notificationController, NotificationType } from '../shared-components/notification/notification';
   import { useZoomImageWheel } from '@zoom-image/svelte';
   import { photoZoomState } from '$lib/stores/zoom-image.store';
+  import { isWebCompatibleImage } from '$lib/utils/asset-utils';
 
   export let asset: AssetResponseDto;
   export let element: HTMLDivElement | undefined = undefined;
 
   let imgElement: HTMLDivElement;
   let assetData: string;
+  let originalDataFetchingTriggered = false;
```

Comment:
I also am not a fan of this name. What about `loadingOriginal`?

Reply:
When this boolean is true, it means that the request is in progress or finished, that's why I tried to find a naming covering these 2 states. What about `isOriginalLoadingOrLoaded`?

Reply:
What about `hasZoomed`? It starts as false and is set to true the first time you zoom in.

Reply:
Good idea!